### PR TITLE
New instrument cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[PYV Instrument Cal CSVs [DT-184]](https://trello.com/c/Eo546k32/184-pyv-instrument-cal-csvs-dt-184)

File: calibration/FLORTD/CGINS-FLORTD-01295__20220909.csv [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+1295,CC_angular_resolution,1.076,Constant; source_file FLORT-D_BBFL2W_SN_1295_Calibration_Files_2022-09-09.zip > BBFL2W-1295.dev; cal date from BBFL2W-1295 Characterization Sheet.pdf
+1295,CC_dark_counts_cdom,48,
+1295,CC_dark_counts_chlorophyll_a,55,
+1295,CC_dark_counts_volume_scatter,52,
+1295,CC_depolarization_ratio,0.039,Constant
+1295,CC_measurement_wavelength,700,Constant
+1295,CC_scale_factor_cdom,9.062E-02,
+1295,CC_scale_factor_chlorophyll_a,1.218E-02,
+1295,CC_scale_factor_volume_scatter,1.826E-06,
+1295,CC_scattering_angle,124,Constant
```
Comment (asmith-whoi) [inline calibration/FLORTD/CGINS-FLORTD-01295__20220909.csv:11]: OK

File: calibration/FLORTD/CGINS-FLORTD-01322__20220912.csv [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+1322,CC_angular_resolution,1.076,Constant; source_file FLORT-D_BBFL2W_SN_1322_Calibration_Files_2022-09-12.zip > BBFL2W-1322.dev; cal date from BBFL2W-1322 Characterization Sheet.pdf
+1322,CC_dark_counts_cdom,50,
+1322,CC_dark_counts_chlorophyll_a,50,
+1322,CC_dark_counts_volume_scatter,51,
+1322,CC_depolarization_ratio,0.039,Constant
+1322,CC_measurement_wavelength,700,Constant
+1322,CC_scale_factor_cdom,8.968E-02,
+1322,CC_scale_factor_chlorophyll_a,1.207E-02,
+1322,CC_scale_factor_volume_scatter,1.817E-06,
+1322,CC_scattering_angle,124,Constant
```
Comment (asmith-whoi) [inline calibration/FLORTD/CGINS-FLORTD-01322__20220912.csv:11]: OK

**Review by asmith-whoi — APPROVED** [review]

File: calibration/DOFSTK/CGINS-DOFSTK-02708__20220702.csv [hunk 3]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-2708,CC_frequency_offset,-8.822879e+002,DOFST-K_SBE43F_SN_2708_Calibration_2022-07-02.zip > 2708.cal
+43-2708,CC_oxygen_signal_slope,3.1931e-004,use Soc(adj) on SOCAdjusted pdf
+43-2708,CC_residual_temperature_correction_factor_a,-4.476314e-003,
+43-2708,CC_residual_temperature_correction_factor_b,2.161883e-004,
+43-2708,CC_residual_temperature_correction_factor_c,-3.437302e-006,
+43-2708,CC_residual_temperature_correction_factor_e,3.600000e-002,
```
[the remark this answers is not in the document]
Comment (batryn) [reply calibration/DOFSTK/CGINS-DOFSTK-02708__20220702.csv:7]: Looks good

File: calibration/FLORTD/CGINS-FLORTD-01003__20220913.csv [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+1003,CC_angular_resolution,1.076,Constant; source_file FLORT-D_BBFL2W_SN_1003_Calibration_Files_2022-09-13.zip > BBFL2W-1003.dev
+1003,CC_dark_counts_cdom,43,
+1003,CC_dark_counts_chlorophyll_a,44,
+1003,CC_dark_counts_volume_scatter,46.175,
+1003,CC_depolarization_ratio,0.039,Constant
+1003,CC_measurement_wavelength,700,Constant
+1003,CC_scale_factor_cdom,8.887E-02,
+1003,CC_scale_factor_chlorophyll_a,1.205E-02,
+1003,CC_scale_factor_volume_scatter,1.835E-06,
+1003,CC_scattering_angle,124,Constant
```
[the remark this answers is not in the document]
Comment (batryn) [reply calibration/FLORTD/CGINS-FLORTD-01003__20220913.csv:11]: Looks good

File: calibration/FLORTD/CGINS-FLORTD-01124__20220927.csv [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+1124,CC_angular_resolution,1.076,Constant; source_file FLORT-D_BBFL2W_SN_1124_Calibration_Files_2022-09-27.zip > BBFL2W-1124.dev
+1124,CC_dark_counts_cdom,47,
+1124,CC_dark_counts_chlorophyll_a,53,
+1124,CC_dark_counts_volume_scatter,50,
+1124,CC_depolarization_ratio,0.039,Constant
+1124,CC_measurement_wavelength,700,Constant
+1124,CC_scale_factor_cdom,9.147E-02,
+1124,CC_scale_factor_chlorophyll_a,1.154E-02,
+1124,CC_scale_factor_volume_scatter,1.586E-06,
+1124,CC_scattering_angle,124,Constant
```
[the remark this answers is not in the document]
Comment (batryn) [reply calibration/FLORTD/CGINS-FLORTD-01124__20220927.csv:11]: Looks good

File: calibration/FLORTD/CGINS-FLORTD-01295__20220909.csv (hunk 1 above)
Comment (batryn) [reply calibration/FLORTD/CGINS-FLORTD-01295__20220909.csv:11]: Looks good

File: calibration/FLORTD/CGINS-FLORTD-01322__20220912.csv (hunk 2 above)
Comment (batryn) [reply calibration/FLORTD/CGINS-FLORTD-01322__20220912.csv:11]: Looks good

File: calibration/FLORTK/CGINS-FLORTK-01117__20220713.csv [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+1117,CC_angular_resolution,1.076,source_file FLORT-K_BBFL2_SN_1117_Calibration_2022-07-13.zip > BBFL2-1117.dev
+1117,CC_dark_counts_cdom,49,
+1117,CC_dark_counts_chlorophyll_a,50,
+1117,CC_dark_counts_volume_scatter,50,
+1117,CC_depolarization_ratio,0.039,
+1117,CC_measurement_wavelength,700,
+1117,CC_scale_factor_cdom,8.860E-02,
+1117,CC_scale_factor_chlorophyll_a,1.211E-02,
+1117,CC_scale_factor_volume_scatter,3.258E-06,
+1117,CC_scattering_angle,124,
```
[the remark this answers is not in the document]
Comment (batryn) [reply calibration/FLORTK/CGINS-FLORTK-01117__20220713.csv:11]: Looks good

File: calibration/PARADK/CGINS-PARADK-20610__20220701.csv [hunk 7]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20610,CC_dark_offset,0.98,source_file PARAD-K_QSP2200_SN_20610_Calibration_2022-07-01.pdf
+20610,CC_scale_wet,9.42E-18,
```
[the remark this answers is not in the document]
Comment (batryn) [reply calibration/PARADK/CGINS-PARADK-20610__20220701.csv:3]: Looks good

**Review by batryn — APPROVED** [review]
Looks good

**Pull request merged**